execution/evm/vm.go

```diff
@@ -493,6 +493,66 @@ func (vm *VM) call(caller acm.Account, callee acm.MutableAccount, code, input []
 			stack.Push64(int64(res))
 			vm.Debugf(" => 0x%X\n", res)
 
+		case SHL: //0x1B
+			shift, popErr := stack.PopU64()
+
+			value := stack.PopBigInt()
+
+			if shift >= 256 || popErr != nil {
```

Comment:
stack.PopU64() might fail because the stack itself underflowed (i.e. there is nothing on the stack, ErrDataStackUnderflow).
What we could do is define a new error in execution/evm/vm.go called something like ErrDataStackTypeInt64Overflow and set that error in vm.PopU64() and vm.Pop64(). Next we can compare the error here like
if shift >= 256 or err == ErrDataStackTypeInt64Overflow {
}
We would still need to return an error in the case that ErrDataStackUnderflow.